services/user.service.js

```diff
@@ -38,8 +38,9 @@ class UserService {
       config.refreshTokenSecret,
       config.refreshTokenExpIn
     )
+    user.password = undefined
 
-    return { accessToken, refreshToken }
+    return { user, accessToken, refreshToken }
```

Comment:
why do you return user there? you don't need this. please use /user with accesstoken endpoint